src/components/CommandBar.js

```diff
@@ -62,10 +62,26 @@ function formatKey(action) {
   return formatKeyShortcut(key);
 }
 
-function debugBtn(onClick, type, className, tooltip) {
+function handlePressAnimation(button) {
+  if (button) {
+    button.style.opacity = "0";
+    button.style.transform = "scale(1.3)";
+    setTimeout(() => {
+      button.style.opacity = "1";
+      button.style.transform = "none";
+    }, 200);
+  }
+}
+
+function debugBtn(onClick, type, className, tooltip, getRefCallback) {
   className = `${type} ${className}`;
   return dom.span(
-    { onClick, className, key: type },
+    {
+      onClick,
+      className,
+      key: type,
+      ref: getRefCallback(type)
```

Comment:
hmm, is it possible to do something like

```js
ref: type
```

and then we can do `this.ref[action]` to get the button in `handleEvent`. 

I would like to avoid adding the buttons to the prototype if we can